engine/org.eclipse.birt.report.engine.emitter.docx/src/org/eclipse/birt/report/engine/emitter/docx/writer/DocxWriter.java

```diff
@@ -66,12 +66,10 @@ public class DocxWriter implements IWordWriter {
 	 * @param compressionMode compression mode
 	 * @param wordVersion     word version
 	 */
-	public DocxWriter(OutputStream out, String tempFileDir, int compressionMode, int wordVersion,
-			boolean wrappedTableHeaderFooter) {
+	public DocxWriter(OutputStream out, String tempFileDir, int compressionMode, int wordVersion) {
```

Comment:
This public method has changed. Can someone get in trouble with it? If so, can you please overload the method?

Reply:
No changes necessary because the latest version is the original version!
With the interface changes I was able to remove my change here.